src/app/player/player.page.ts

```diff
@@ -137,72 +137,67 @@ export class PlayerPage implements OnInit, OnDestroy, PlayerActionHandlerDelegat
     });
   }
   async ionViewWillEnter() {
-    const playerInterval = setInterval(() => {
```

Comment:
@kiranharidas187  setTimeOut was introduced to resolve one of the issue. We cant remove setTimeout